docs/pages/connect-your-client/gui-clients.mdx

```diff
@@ -69,36 +69,21 @@ Ensure that your environment includes the following:
 
 <ScopedBlock scope={["oss", "enterprise"]}>
```

Comment:
Since users tend not to slick the scope switcher at the top of the page, we should use `Tabs` instead of a `ScopedBlock`.